### PR TITLE
Fix users import when an entry in /etc/shadow is missing

### PR DESCRIPTION
**Pull request by imobach**
* Fixes [bsc#978648](https://bugzilla.suse.com/show_bug.cgi?id=978648).

### CODE REVIEW

File: package/yast2-users.changes [hunk 1]
```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Thu May  5 12:32:39 UTC 2016 - igonzalezsosa@suse.com
+
+- Fix users import when an entry in /etc/shadow is missing
```
Comment (ancorgs) [inline package/yast2-users.changes]: NP: In fact it's when ALL entries for local users are missing.

**Comment by ancorgs** [comment]
LGTM

File: package/yast2-users.changes (hunk 1 above)
Comment (imobach) [inline package/yast2-users.changes]: I've updated the message but using `non-system` instead of `local` (although we use the term `local` in the code) :)

**Comment by imobach** [comment]
Thanks!

**Pull request merged**